<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortAssistant</class>
 <widget class="QWidget" name="SerialPortAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialPortAssistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/images/spnew.png</normaloff>:/images/spnew.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>541</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="displayGrpBox">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Display</string>
      </property>
      <widget class="QGroupBox" name="inputGrpBox">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>300</y>
         <width>521</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Input</string>
       </property>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>491</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>69</y>
          <width>491</width>
          <height>29</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="hexSendCheckBox">
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>HexSend</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sendBtn">
           <property name="text">
            <string>Send</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/roadControl64x64.png</normaloff>:/images/roadControl64x64.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>26</y>
         <width>401</width>
         <height>192</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>411</width>
         <height>28</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="rxCountLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>RX:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="txCountLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TX:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber_2">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="hexDisplayCheckBox">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>ToHex</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <zorder>textEdit</zorder>
      <zorder>layoutWidget</zorder>
      <zorder>inputGrpBox</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>311</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="configGrpBox">
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>251</width>
         <height>205</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="portCombox">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Baud</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="baudCombox">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string>57600</string>
          </property>
          <item>
           <property name="text">
            <string>1200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>19200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>38400</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>57600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>115200</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>DataBits</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="dataCombox">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string>8</string>
          </property>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Parity</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="parityCombox">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Even</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Odd</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Space</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mark</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>StopBit</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="stopCombox">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1.5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PreRead</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_2">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="refreshBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>261</y>
         <width>113</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Refresh   </string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/refresh128x128.png</normaloff>:/images/refresh128x128.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="connectBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>298</y>
         <width>113</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Connect   </string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/connect64x64.png</normaloff>:/images/connect64x64.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="disconnectBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>335</y>
         <width>113</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>DisConnect</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/close64x64.png</normaloff>:/images/close64x64.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="clearBtn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>370</y>
         <width>113</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear     </string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/images/trash64x64.png</normaloff>:/images/trash64x64.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
